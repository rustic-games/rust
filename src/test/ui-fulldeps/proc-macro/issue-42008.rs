// aux-build:issue-42008.rs

#![feature(proc_macro_non_items)]

#[macro_use]
extern crate issue_42008;

fn main() {
    let _ = foo!("");
}
