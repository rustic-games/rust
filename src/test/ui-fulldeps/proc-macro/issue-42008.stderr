error: missing documentation for a module
   |
note: lint level defined here
  --> $DIR/auxiliary/issue-42008.rs:4:9
   |
LL | #![deny(missing_docs)]
   |         ^^^^^^^^^^^^

error: missing documentation for a function

error: aborting due to 2 previous errors

